docs/web-sdk/transcribing-live-audio-through-microphone.md

```diff
@@ -0,0 +1,117 @@
+---
```

Comment:
This file should be deleted. It contains Web SDK details which we are not pushing right now (as mentioned earlier).